src/renderer/helpers/api/local.js

```diff
@@ -1692,3 +1692,25 @@ export async function getHashtagLocal(hashtag) {
   const innertube = await createInnertube()
   return await innertube.getHashtag(hashtag)
 }
+
+export async function getCommunityPostLocal(postId, channelId) {
+  const innertube = await createInnertube()
+  if (channelId == null) {
+    const resolved = await innertube.resolveURL('https://www.youtube.com/post/' + postId)
+    channelId = resolved.payload.browseId
+  }
+
+  const postPage = await innertube.GetPost(postId, channelId)
+  return parseLocalCommunityPost(postPage.posts.first())
+}
+
+/**
+ * @param {string} postId
+ * @param {string} channelId
+ * @param {boolean} sortByNewest
+ */
+export async function getCommunityPostComments(postId, channelId, sortByNewest) {
```

Comment:
Could you please include local in the name, like we do for other functions that this file exports.